packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsFilesForm.tsx

```diff
@@ -80,58 +126,20 @@ export default function DataAccessRequestAccessorsFilesForm(
     researchProjectId,
     onCancel,
   } = props
-  const { data: user } = useGetCurrentUserProfile()
+  const { accessToken } = useSynapseContext()
+  const isLoggedIn = Boolean(accessToken)
+  const { data: user } = useGetCurrentUserProfile({ enabled: isLoggedIn })
```

Comment:
Ensure the user is logged in before we fetch the profile data (edge case that I encountered in Storybook, probably not possible in the wild)